bitnami/thanos/README.md

```diff
@@ -109,507 +109,542 @@ The following tables lists the configurable parameters of the Thanos chart and t
-| `query.ingress.grpc.enabled`                     | Enable ingress controller resource (GRPC)                                                                                                                     | `false`                         |
-| `query.ingress.grpc.certManager`                 | Add annotations for cert-manager (GRPC)                                                                                                                       | `false`                         |
-| `query.ingress.grpc.hostname`                    | Default host for the ingress resource (GRPC)                                                                                                                  | `thanos.local`                  |
-| `query.ingress.grpc.annotations`                 | Ingress annotations (GRPC)                                                                                                                                    | `[]`                            |
-| `query.ingress.grpc.extraHosts[0].name`          | Additional hostnames to be covered (GRPC)                                                                                                                     | `nil`                           |
-| `query.ingress.grpc.extraHosts[0].path`          | Additional hostnames to be covered (GRPC)                                                                                                                     | `nil`                           |
-| `query.ingress.grpc.extraTls[0].hosts[0]`        | TLS configuration for additional hostnames to be covered (GRPC)                                                                                               | `nil`                           |
-| `query.ingress.grpc.extraTls[0].secretName`      | TLS configuration for additional hostnames to be covered (GRPC)                                                                                               | `nil`                           |
-| `query.ingress.grpc.secrets[0].name`             | TLS Secret Name (GRPC)                                                                                                                                        | `nil`                           |
-| `query.ingress.grpc.secrets[0].certificate`      | TLS Secret Certificate (GRPC)                                                                                                                                 | `nil`                           |
-| `query.ingress.grpc.secrets[0].key`              | TLS Secret Key (GRPC)                                                                                                                                         | `nil`                           |
+| Parameter                                                 | Description                                                                                                                                                   | Default                         |
```

Comment:
It is really difficult to see what changed here, could you just edit the needed rows please?

Reply:
@miguelaeh will do that.